android/java/org/chromium/chrome/browser/upgrade/BraveUpgradeJobIntentServiceImpl.java

```diff
@@ -47,22 +45,15 @@ public void onSuccess() {
                                             BraveRewardsNativeWorker braveRewardsNativeWorker =
                                                     BraveRewardsNativeWorker.getInstance();
                                             if (braveRewardsNativeWorker != null
-                                                    && braveRewardsNativeWorker.isSupported()
-                                                    && BravePrefServiceBridge.getInstance()
-                                                            .getSafetynetCheckFailed()) {
+                                                    && braveRewardsNativeWorker.isSupported()) {
```

Comment:
I don't think we need `BraveUpgradeJobIntentServiceImpl` class at all. The whole it's purpose was to re-perform the SafetyNet check

Reply:
i am going to remove in a follow up commit.